pkg/kubelet/containerdshim/containerd_container.go

```diff
@@ -162,38 +173,54 @@ func (cs *containerdService) CreateContainer(podSandboxID string, containerConfi
 
 	// mikebrow TODO proper console handling
 	glog.V(2).Infof("CreateContainer for container %s container directory %s", containerID, containerDir)
-	response, err := cs.cdClient.Create(gocontext.Background(), create)
+	response, err := cs.containerService.Create(gocontext.Background(), create)
 	if err != nil {
 		return "", err
 	}
 
+	containerStoreLock.Lock()
```

Comment:
note: creating state, would necessitate the containerStore allocation to occur above the create call.